stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/EventHandlerImpl.kt

```diff
@@ -465,4 +458,12 @@ internal class EventHandlerImpl(
             queryRepo.handleEvents(sortedEvents)
         }
     }
+
+    private fun Message.enrichWithOwnReactions(batch: EventBatchUpdate, user: User) {
```

Comment:
The logic behind this:
If event is from current user then we are fine with `ownReactions` coming from event.
If event is from other user, then we replace other user's `ownReactions` with our own `ownReactions` from the database